media-libs/libplacebo/metadata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>gentoo@haasn.xyz</email>
+		<name>Niklas Haas</name>
+	</maintainer>
```

Comment:
If you'd like to maintain this via proxy-maint, please add us as well (below yourself). Otherwise, you'd need some other committer.

Reply:
I've added proxy-maint as well.